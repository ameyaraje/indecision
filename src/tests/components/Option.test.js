import React from 'react';
import { shallow } from 'enzyme';

import Option from '../../components/Option';

test('should render Option component', () => {
    const wrapper = shallow(<Option />);

    expect(wrapper).toMatchSnapshot();
});