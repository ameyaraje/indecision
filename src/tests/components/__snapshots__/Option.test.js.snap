// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Option component 1`] = `
<div
  className="option"
>
  <p
    className="option__text"
  />
  <button
    className="button button--link"
    onClick={[Function]}
  >
    Remove
  </button>
</div>
`;
